apiclient/types/mcpcatalog.go

```diff
@@ -3,14 +3,12 @@ package types
 type MCPCatalog struct {
 	Metadata
 	MCPCatalogManifest
-	IsReadOnly bool `json:"isReadOnly,omitempty"`
```

Comment:
Now that there is only one catalog, we don't need to distinguish between read-only and not-read-only.